application/src/main/java/com/knubisoft/application/InitMongoDB.java

```diff
@@ -0,0 +1,31 @@
+package com.knubisoft.application;
+
+import com.knubisoft.application.model.Developer;
+import com.knubisoft.application.repository.DeveloperRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class InitMongoDB {
+    private DeveloperRepository repository;
+
+    @Autowired
+    public InitMongoDB(DeveloperRepository repository) {
+        this.repository = repository;
+    }
+
+    @PostConstruct
+    private void init() {
```

Comment:
Maybe would be better to move this to init file for docker container instead of leaving this code here, if we don't want our base to be empty every time we raise it up.

Reply:
yes, this is a good idea, but i guess in our project this data in near future will be deprecated and we wont need them, so for them, i believe, it is unnecessary

Reply:
this is just representation that connection to mongo is actually working